test/dlc_tests/ecs/tensorflow/inference/test_ecs_tensorflow_inference.py

```diff
@@ -26,6 +26,27 @@ def test_ecs_tensorflow_inference_cpu(tensorflow_inference, ecs_container_instan
         ecs_utils.tear_down_ecs_inference_service(ecs_cluster_arn, service_name, task_family, revision)
 
 
+@pytest.mark.parametrize("ecs_instance_type", ["c5.4xlarge"], indirect=True)
+@pytest.mark.parametrize("ecs_ami", [ECS_AML2_CPU_USWEST2], indirect=True)
+@pytest.mark.parametrize("ei_accelerator_type", ["eia1.large"], indirect=True)
+def test_ecs_tensorflow_inference_eia(tensorflow_inference_eia, ecs_container_instance, region, eia_only):
+    worker_instance_id, ecs_cluster_arn = ecs_container_instance
+    public_ip_address = ec2_utils.get_public_ip(worker_instance_id, region=region)
+
+    model_name = "saved_model_half_plus_two"
+    service_name = task_family = revision = None
+    try:
+        service_name, task_family, revision = ecs_utils.setup_ecs_inference_service(
+            tensorflow_inference_eia, "tensorflow", ecs_cluster_arn, model_name, worker_instance_id, region=region, ACCELERATOR_TYPE="eia1.large",
+        )
```

Comment:
The `accelerator_type` input to this function should come from the `ei_accelerator_type` fixture.